spp_programs_auto_enroll/models/programs.py

```diff
@@ -0,0 +1,32 @@
+# Part of OpenSPP. See LICENSE file for full copyright and licensing details.
+
+import logging
+from datetime import timedelta
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class CustomG2PProgram(models.Model):
+    _inherit = "g2p.program"
+
+    auto_enroll = fields.Boolean(default=False)
```

Comment:
```suggestion
    auto_enroll = fields.Boolean()
```
boolean field default is False